packages/addon-dev/src/rollup.ts

```diff
@@ -59,8 +59,8 @@ export class Addon {
     return hbs(options);
   }
 
-  gjs(options?: { inline_source_map: boolean }) {
```

Comment:
removing this makes this PR a breaking change... _but_ inline source map is likely never something anyone wants as it hasn't been compatible with rollup.

So... this PR "could" be a bugfix, I suppose.

Reply:
@NullVoxPopuli IMO it;s change in public API - so breaking regardless if ppl prefer/use inline source maps

Reply:
@SergeAstapov agreed, and integers are cheap 👍